kuma/static/styles/components/codepen-iex/codepen-iex.scss

```diff
@@ -0,0 +1,21 @@
+@import '../../includes/vars';
+@import '../../includes/mixins';
+
+.codepen-iex {
+    position: relative;
+
+    .tryit {
+        position: absolute;
+        top: 10px;
+        right: 10px;
+        background-color: #fff;
+        border: 3px solid #3d7e9a;
+        height: 2rem;
+        font-size: 1rem;
+        cursor: pointer;
+
+        &:hover {
+            @include focus-visible();
```

Comment:
I'm not sure I agree with using a focus style for the hover state. Should we define a separate thing for hover, so that we can later decide to use different styling for focus and hover?

Also, if you're doing :hover, is it worth doing :active as well? Or does the <button> do something reasonable by default here?

Reply:
Agreed. As this is an early prototype, I was a little lazy here ;p I am going to not be, and create a more effective `:hover/:focus/:active` style we can adopt for situations like these. Thanks for calling it out @davidflanagan